library/core/talaiot/src/main/kotlin/io/github/cdsap/talaiot/extensions/String.kt

```diff
@@ -22,3 +22,25 @@ fun String.toBytes(): String? {
     }
     return null
 }
+
+fun String.gradleVersionCompatibleWithIsolatedProjects(): Boolean {
```

Comment:
Tip:

```kotlin
val GradleVersion.isCompatibleWithIsolatedProjects: Boolean =
    GradleVersion.version("8.5") <= this.baseVersion
```
and
```patch
-GradleVersion.current().version.gradleVersionCompatibleWithIsolatedProjects()
+GradleVersion.current().isCompatibleWithIsolatedProjects
```

Notables:
 * GradleVersion is comparable
 * baseVersion strips the snapshot/rc build part, leaving you with only the "major.minor.patch" part